resolwe/flow/views.py

```diff
@@ -200,20 +202,13 @@ def create(self, request, *args, **kwargs):
         # translate processe's slug to id
         process_slug = request.data.get('process', None)
         process_query = Process.objects.filter(slug=process_slug)
+        process_query = get_objects_for_user(request.user, 'view_process', process_query)
         if not process_query.exists():
-            # XXX: security - is it ok to reveal which processes (don't) exist?
```

Comment:
Before there were different status codes (401, 403, 404) indicating if process exists or not. Now you get 404 if you don't have the permission or if the process doesn't exist.